src/ert/validation/runpath_argument.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from .argument_definition import ArgumentDefinition
+from .validation_status import ValidationStatus
+
+
+class RunPathArgument(ArgumentDefinition):
+    INVALID_PATH = "The specified runpath does not exist."
+    MISSING_PERMISSION = "You are missing permissions for the specified runpath."
+
+    def __init__(self, **kwargs: bool) -> None:
+        super().__init__(**kwargs)
+
+    def validate(self, token: str) -> ValidationStatus:
+        parsed_runpath_without_suffix = "/".join(token.split("/")[:-2])
+        validation_status = super().validate(token)
+        try:
+            os.listdir(parsed_runpath_without_suffix)
```

Comment:
Nitpicking here, but is it necessary to list the dir or  could we use `os.path.isdir` and `os.access` instead?
Something like this:

```python
if not os.path.isdir(parsed_runpath_without_suffix):
    validation_status.setFailed()
    validation_status.addToMessage(RunPathArgument.INVALID_PATH)
elif not os.access(parsed_runpath_without_suffix, os.R_OK):
    validation_status.setFailed()
    validation_status.addToMessage(RunPathArgument.MISSING_PERMISSION)
else:
    validation_status.setValue(token)
```

Should be more efficient and perhaps easier to read.

Reply:
listdir both checks if the dir exists and if the user has access. I can change it to be more concrete 👌.
Do you think the user only needs read_access, or should I check all rwx?

Reply:
I think users need only have read access to load results manually. Might be wrong though.

Reply:
I tested it manually, we apparently need r-x permissions. Will update it right away

Reply:
Should I test for read permission first, then execute permission? I could have different error messages for those two, or is that excessive?

Reply:
I think a single error message is enough.